### PR TITLE
docs: Zerho is no more

### PR DESCRIPTION
**Pull request by suhr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,8 +92,6 @@ Right now we are working on the client part.
 
 tox-rs has contributions from many users. See [AUTHORS.md](/AUTHORS.md). Thanks everyone!
 
-tox-rs is sponsored by [Zerho LLC](https://zerho.info/).
```
Comment (kpp) [inline README.md]: was sponsored?*

Comment (suhr) [reply README.md]: Let it go.

Comment (kpp) [reply README.md]: See our LICENSE-MIT

Comment (suhr) [reply README.md]: Now I tweaked it a bit. I'm not sure if removing Zerho from the license is safe, so I kept it.

**Pull request merged**